ansible-ipi-install/group_vars/all.yml

```diff
@@ -46,3 +46,14 @@ worker_count: 0
 jumbo_mtu: false
 # set to true only if you requested a public routable VLAN for your cloud in scale lab
 routable_api: false
+
+
+#post-install
+post_install: "{{ lookup('env', 'POST_INSTALL')|default(true, true) }}"
```

Comment:
I think we should default this variable to false to prevent modifying the current behavior

Reply:
ack, will switch it back to true after this gets merged for my playground